libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js

```diff
@@ -9,26 +9,24 @@ const { exec } = require('child_process');
 // Note: This file is intentionally not named *.test.js to ensure it isn't run
 // via `yarn run test` or `npm run test`.
 
+const maybeIt = process.platform === 'win32' ? it : it.skip;
+
 async function runCommand(command) {
-    try {
-        return new Promise((resolve, _reject) => {
-            exec(
-                command,
-                {
-                    cwd: __dirname,
-                },
-                (err, stdout, stderr) => {
-                    if (err || stderr) {
-                        console.error(err ?? stderr);
-                    }
-                    resolve({ error: err ?? stderr, log: stdout });
+    return new Promise((resolve, reject) => {
+        exec(
+            command,
+            {
+                cwd: __dirname,
+            },
+            (err, stdout, _stderr) => {
+                // Ignore stderr since npm warnings get routed there.
+                if (err) {
+                    reject(err);
```

Comment:
You'll want to return or move `resolve` into an `else` block since you aren't allowed to invoke both.